### PR TITLE
Licensing update

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[843161f](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/843161fc6739d3f2eefa25e123536a352cc5ce96)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/New-MsolUser.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/New-MsolUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/new-msoluser?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/Set-MsolUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoluserlicense?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0-preview&branch=pr-en-us-806) (azureadps-2.0-preview) |
[azureadps-2.0/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0&branch=pr-en-us-806) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C14%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204140921525667-806%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-15T09%3A23%3A10Z&sr=b&sp=r&sig=fVRSiS0cCjxLKNfxEjw2q1IG6Yhyd%2FtwlcI1Ii13D1c%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[83c4381](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/83c4381d8d1f1308e5095cf5e5e8cb8af268183e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/New-MsolUser.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/New-MsolUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/new-msoluser?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/Set-MsolUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoluserlicense?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0-preview&branch=pr-en-us-806) (azureadps-2.0-preview) |
[azureadps-2.0/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0&branch=pr-en-us-806) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C14%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204140951336574-806%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-15T09%3A53%3A32Z&sr=b&sp=r&sig=JDVt9ctU6Uhasrsb9C7Wt2F%2BB9y2On5oNa91GDinBhk%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
@CelesteDG, this is ready for your review following Merill's go ahead. Thanks.

File: azureadps-1.0/MSOnline/New-MsolUser.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ The user does not have any licenses assigned.
 A random password is generated for the user.
 
 ### Example 2: Create a user and assign a license
+
+>[!NOTE]
+> The **-LicenseAssignment** parameter is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For additional information, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Graph API reference page.
```
Comment (CelesteDG) [inline azureadps-1.0/MSOnline/New-MsolUser.md:50]:
```suggestion
> The **-LicenseAssignment** parameter is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For more info, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Microsoft Graph API.
```

File: azureadps-1.0/MSOnline/New-MsolUser.md [hunk 2]
```diff
@@ -244,6 +247,9 @@ Accept wildcard characters: False
 ### -LicenseAssignment
 Specifies an array of licenses to assign the user.
 
+>[!NOTE]
+> The **-LicenseAssignment** parameter is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For additional information, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Graph API reference page.
```
Comment (CelesteDG) [inline azureadps-1.0/MSOnline/New-MsolUser.md:251]:
```suggestion
> The **-LicenseAssignment** parameter is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For more info, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Microsoft Graph API.
```

File: azureadps-1.0/MSOnline/Set-MsolUserLicense.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ ms.custom: iamfeature=PowerShell
 ## SYNOPSIS
 Updates the license assignment for a user.
 
+>[!NOTE]
+> The **Set-MsolUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For additional information, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Graph API reference page.
```
Comment (CelesteDG) [inline azureadps-1.0/MSOnline/Set-MsolUserLicense.md:16]:
```suggestion
> The **Set-MsolUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For more info, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Microsoft Graph API.
```

File: azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md [hunk 4]
```diff
@@ -13,6 +13,9 @@ schema: 2.0.0
 ## SYNOPSIS
 Adds or removes licenses for a Microsoft online service to the list of assigned licenses for a user.
 
+>[!NOTE]
+> The **Set-AzureADUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For additional information, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Graph API reference page.
```
Comment (CelesteDG) [inline azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md:17]:
```suggestion
> The **Set-AzureADUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For more info, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Microsoft Graph API.
```

File: azureadps-2.0/AzureAD/Set-AzureADUserLicense.md [hunk 5]
```diff
@@ -10,6 +10,9 @@ schema: 2.0.0
 ## SYNOPSIS
 Adds or removes licenses for a Microsoft online service to the list of assigned licenses for a user.
 
+>[!NOTE]
+> The **Set-AzureADUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For additional information, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Graph API reference page.
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/Set-AzureADUserLicense.md:14]:
```suggestion
> The **Set-AzureADUserLicense** cmdlet is deprecated. Learn [how to assign licenses with Microsoft Graph PowerShell](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell). For more info, see the [Assign License](/graph/api/user-assignlicense?view=graph-rest-1.0&tabs=powershell) Microsoft Graph API.
```

**Review by CelesteDG — CHANGES_REQUESTED** [review]
Simplified

**Comment by billmath** [comment]
Docs Build status updates of commit _[f1cffc5](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/f1cffc54897e585e542438b29a664cb88784ec72)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/New-MsolUser.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/New-MsolUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/new-msoluser?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-1.0/MSOnline/Set-MsolUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoluserlicense?view=azureadps-1.0&branch=pr-en-us-806) (azureadps-1.0) |
[azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0-preview/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0-preview&branch=pr-en-us-806) (azureadps-2.0-preview) |
[azureadps-2.0/AzureAD/Set-AzureADUserLicense.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/licensing-update/azureadps-2.0/AzureAD/Set-AzureADUserLicense.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/set-azureaduserlicense?view=azureadps-2.0&branch=pr-en-us-806) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C27%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204271646378552-806%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-28T16%3A48%3A25Z&sr=b&sp=r&sig=l09lzk5ukd3nUQTk5ZGDH%2FaEamIOpQlHpzwRw%2BhF9XA%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Thank you, @CelesteDG. I have committed your suggestions.

**Pull request merged**